### ISSUE TITLE
New time intervals

### ISSUE THREAD

**Issue by Jan02** [micz/ThunderStats#84]
I think there should also be these intervals:
* Last two weeks
* Last month
* Last quarter
* Last 6 months
* Last year

**Comment by micz** [comment]
In version 1.1 will be added a custom date interval tab, see #67.
In version 1.0 will be possible to choose the number of days for the current "7 days" tab. This has been already implemented in the current source. See 3ddc665570a66bf19493c635a3f2b3ef34aae8f0.
Using 30 or more days Thunderbird will work a bit to crunch the data (it depends on your PC and how many mails you send and receive).
What you are asking, with those big time intervals, could be implemented only after some kind of data aggregation, to speed up the system, see #9.

**Comment by micz** [comment]
The new custom date interval tab (#67) has been implemented and will be released with version 1.1.

This request has been added to the 1.1 milestone and it will be implemented as "shortcuts" that will modify the date interval in the custom date interval tab.

**Comment by Jan02** [comment]
I forgot, the "ALL" time interval should be there too, with a warning it may take a while for the index to build.

**Comment by micz** [comment]
The feature has been implemented, see the screeshot.

![screen](https://cloud.githubusercontent.com/assets/61795/8992972/323b32d6-3702-11e5-8119-9dd9bcb461d1.png)

I've added, for this version, these bookmarks: current week, last week, current month, last month.
Any bigger interval is really too big and I don't what to put them in a bookmark, but an user could manually choose any kind of interval.

![screenbookmark](https://cloud.githubusercontent.com/assets/61795/8992988/5f31d81c-3702-11e5-9bae-e397a2a2b9fc.png)
